whois-nrtm/src/test/java/net/ripe/db/whois/nrtm/integration/NrtmClientACLLimitTestIntegration.java

```diff
@@ -0,0 +1,103 @@
+package net.ripe.db.whois.nrtm.integration;
+
+import com.jayway.awaitility.Awaitility;
+import com.jayway.awaitility.Duration;
+import net.ripe.db.whois.common.IntegrationTest;
+import net.ripe.db.whois.common.dao.jdbc.DatabaseHelper;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.common.source.SourceContext;
+import net.ripe.db.whois.nrtm.NrtmServer;
+import net.ripe.db.whois.nrtm.client.NrtmImporter;
+import net.ripe.db.whois.query.acl.AccessControlListManager;
+import net.ripe.db.whois.query.acl.IpResourceConfiguration;
+import net.ripe.db.whois.query.support.TestPersonalObjectAccounting;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.net.InetAddress;
+
+@Category(IntegrationTest.class)
```

Comment:
Can you add one "success" test, where the client *isn't* blocked or denied by ACL?

Reply:
Have added a success case in junit. 

Actually this integration test does not make sense here. As we are just waiting for 5 seconds and checking the nrtm client thread count > 0, (copied from NrtmClientInvalidHostTestIntegration). We should be deleting this integration test and same for NrtmClientInvalidHostTestIntegration or find a different way to check if channel is open or not.  

Infact NrtmClientTestIntegration also does not do much or  am I missing something?

Reply:
for NrtmClientTestIntegration  we inject different jdbctemplate that is fine then

Reply:
Thanks for fixing the tests